### ISSUE TITLE
Openshift Binaries v 1.1(.0.1) doesn't deploy router and docker registry

### ISSUE THREAD

**Issue by charliemaiors** [openshift/origin#6632]
Hi, after this [issue](https://github.com/openshift/origin/issues/6568) I've upgraded Openshift first on my VM and after I've decided to upgrade it on a physical machine replicating the steps (that consist in wiping all Openshift Docker images from the local machine, replace the binaries, wipe the openshift.local.* folders saving the user database) and restart everything; on my VM everything works fine, but on the physical host Openshift doesn't deploy the docker registry and the ha proxy router.
I've to pull manually the images and restart/redeploy openshift?

I've followed this [guide](https://docs.openshift.org/latest/install_config/install/prerequisites.html) from prerequisites using the quick installation

**Comment by pweil-** [comment]
You should not have to manually pull the images.  What state do you end up in when the deployment fails?  Is there any indication in the logs for the deployment of the registry/router or events in the namespace that they're deployed in that indicate there was a failure pulling the image?

**Comment by charliemaiors** [comment]
There is no indication, just remaining in pending state. Could be an issue with docker storage (that I've left with default option and I've ran a lot of containers on that machine)?

**Comment by charliemaiors** [comment]
I've reinstalled CentOS, installed and configured all the packages in prerequisites Origin installation guide and configured Docker storage (with default options using docker-storage-setup command), reinitialized docker, downloaded again the binaries, started openshift and tried to deploy the docker registry and ha proxy router but still nothing.
The logs from the pods using ```bash oc logs docker-registry-1-deploy``` gives no clue. The docker images are not pulled. Openshift log file (I've ran openshift using ```#openshift start &> openshift.log &```) is attached
[openshift-log.docx](https://github.com/openshift/origin/files/91967/openshift-log.docx)
